app/src/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -116,7 +117,7 @@ public void onClick(View v) {
         addQuestionText(p);
         addHelpPlaceholder(p);
         addHintText(p);
-
+        ACRAUtil.addCustomData(ACRAUtil.LAST_QUESTION_PROMPT, mPrompt.getLongText());
```

Comment:
as a heads up, question widgets can be created without being the last displayed prompt. The activity setting the view is really the only way we know.